selfdrive/locationd/locationd.cc

```diff
@@ -326,7 +326,7 @@ void Localizer::handle_gps(double current_time, const cereal::GpsLocationData::R
   VectorXd ecef_pos = this->converter->ned2ecef({ 0.0, 0.0, 0.0 }).to_vector();
   VectorXd ecef_vel = this->converter->ned2ecef({ log.getVNED()[0], log.getVNED()[1], log.getVNED()[2] }).to_vector() - ecef_pos;
   float ecef_pos_std;
-  if (ublox_available) {
+  if (gnss_source == LocalizerGnssSource::UBLOX) {
```

Comment:
Actually while you're at it, fix this with configure_gnss_source too? You can define a variance_factor and vertical_variance factor, and set them to 0,1,3 appropriately.